target/linux/realtek/dts-5.10/rtl8393_netgear_gs750e.dts

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl839x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "netgear,gs750e", "realtek,rtl8393-soc";
+	model = "Netgear GS750E";
+
+	aliases {
+		label-mac-device = &ethernet0;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	virtual_flash {
+		compatible = "mtd-concat";
+
+		devices = <&fwconcat0 &fwconcat1 &fwconcat2>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "firmware";
+				reg = <0x0 0x0>;
+				compatible = "openwrt,uimage", "denx,uimage";
+				openwrt,ih-magic = <0x174e4741>;
+			};
+		};
+	};
+};
+
+&gpio0 {
+	system_reset {
+		gpio-hog;
+		line-name = "System Reset PIN";
+		gpios = <&gpio0 10 GPIO_ACTIVE_LOW>;
+		output-low;
+	};
+};
+
+&spi0 {
+	status = "okay";
```

Comment:
Small nit: add a newline between "status" and "flash@0".

Reply:
I will fix it, thanks!